docs/content/users/quickstart.md

```diff
@@ -250,8 +249,7 @@ While the generic `php` project type is [ready to go](./project.md) with any CMS
             mkdir my-statamic-app
             cd my-statamic-app
             ddev config --project-type=laravel --docroot=public --create-docroot
-            ddev composer create --prefer-dist --no-install --no-scripts statamic/statamic
-            ddev composer install
+            ddev composer create --prefer-dist statamic/statamic -y
             ddev exec "php artisan key:generate"
```

Comment:
```suggestion
            ddev composer create --no-install --prefer-dist statamic/statamic -y
            ddev composer run-script post-root-package-install
            ddev composer install
            ddev composer run-script post-create-project-cmd
            ddev php please make:user
```

Reply:
```suggestion
            ddev composer create --prefer-dist statamic/statamic -y
            ddev composer run-script post-create-project-cmd
            ddev php please make:user --super
```

Reply:
I would leave out the --no-install flag so that the user doesn't have to run the `composer install` since the pull request introduces convenience of not needing the --no-install flag anymore. so one command less to enter.  and the other two i've picked from your previous suggestion. in regards of the make:user `--super` i haven't found that something like that existed. that also saves my previous question in regards of the comment. that isn't necessary anymore that way.

Reply:
Will come up with a better solution, was only playing around here.